README.md

```diff
@@ -43,17 +43,20 @@ Registered entry points for aiida.calculations:
 * core.transfer
 * janus.opt
 * janus.sp
+* janus.md
 ```
 
 
 ## Usage
 
-A quick demo of how to submit a calculation using the provided example files submit_singlepoint.py and submit_geomopt.py:
+A quick demo of how to submit a calculation using the provided example files:
 ```shell
 verdi daemon start     # make sure the daemon is running
 cd examples/calculations
-verdi run submit_singlepoint.py "janus@localhost" --architecture mace --model "/path/to/model"    # run test calculation
-verdi run submit_geomopt.py "janus@localhost" --structure "path/to/structure" --model "path/to/model" --steps 5 --fully-opt True
+verdi run submit_singlepoint.py "janus@localhost" --architecture mace --model "/path/to/model"    # run singlepoint calculation
```

Comment:
Does this not need a structure?

Reply:
In theory there are default values for everything, so it does not necessarily need a structure. but I'll add it, I don't know why I removed it